src/indexOf.js

```diff
@@ -21,6 +21,7 @@ var _isArray = require('./internal/_isArray');
  *
  *      R.indexOf(3, [1,2,3,4]); //=> 2
  *      R.indexOf(10, [1,2,3,4]); //=> -1
+ *
```

Comment:
There's no guide, I'm sorry! Hover over the emoji, though, and you will see its name – `:space_invader:`, in this case. ;)